src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -259,4 +260,90 @@ public void systemEndsPatientVisit() {
 		assertNull(visitsDashboardPage.getActiveVisit());
 		dashboardPage = visitsDashboardPage.goToPatientDashboard();
 	}
+
+	@When("a user clicks on Allergy2 link from Patient dashboard page2")
+	public void loadManageAllergy2() {
+		allergyPage = (AllergyPage) dashboardPage.clickOnAllergiesWidgetLink()
+				.waitForPage();
+	}
```

Comment:
You can make re-use of [loadAllergiesPage()](https://github.com/openmrs/openmrs-contrib-qaframework/blob/2181d1a931c69a264d3c4c6e7e159d4ecf60f9aa/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java#L113) instead than creating a new method that achieves the same goal.

Reply:
@Ndacyayisenga-droid did you see the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/137/files#r722437930)?